src/equations/2d/compressible_euler.jl

```diff
@@ -806,18 +806,20 @@ end
 # Determine maximum stable time step based on polynomial degree and CFL number
 function calc_max_dt(u, element_id, invjacobian, cfl,
                      equation::CompressibleEulerEquations2D, dg)
-  λ_max = 0.0
+  λ_max1 = 0.0
+  λ_max2 = 0.0
   for j in 1:nnodes(dg), i in 1:nnodes(dg)
     rho, rho_v1, rho_v2, rho_e = get_node_vars(u, dg, i, j, element_id)
     v1 = rho_v1 / rho
     v2 = rho_v2 / rho
     v_mag = sqrt(v1^2 + v2^2)
     p = (equation.gamma - 1) * (rho_e - 1/2 * rho * v_mag^2)
```

Comment:
```suggestion
    p = (equation.gamma - 1) * (rho_e - 1/2 * rho * (v1^2 + v2^2))
```
That's different in Taam/Taal